contracts/contracts/validator/Mock_Uptime_feed.cairo

```diff
@@ -0,0 +1,53 @@
+%lang starknet
```

Comment:
Let's separate `cairo` and `solidity` contracts into two separate subfodlers:

- `contracts/contracts/cairo`
- `contracts/contracts/solidity`